app/controllers/pxe_controller/iso_datastores.rb

```diff
@@ -23,7 +23,7 @@ def iso_datastore_edit
   end
 
   def iso_datastore_create
-    assert_privileges('iso_datastore_create')
+    assert_privileges('iso_datastore_new')
```

Comment:
I think you need to add a pxe server and try to add an iso datastore to it.  I couldn't find what I needed in the DB to recreate this in the UI but the routes_spec should recreate it on it's own.

Reply:
@bdunne Do you remember?

Reply:
Again, @kavyanekkalapu found it... 🎉 

![image](https://user-images.githubusercontent.com/19339/118665946-c31d1900-b7c0-11eb-9221-909fec3bc601.png)

Reply:
I think that was implemented for RHEV/oVirt, so you probably need one of those providers